py4j-java/src/main/java/py4j/ClientServer.java

```diff
@@ -160,4 +160,85 @@ public Object getPythonServerEntryPoint(Class[] interfacesToImplement) {
 		return proxy;
 	}
 
+	/**
+	 * Helper class to make it easier and self-documentating how a
+	 * {@link ClientServer} is constructed.
+	 */
+	public static class ClientServerBuilder {
+		private int fJavaPort;
```

Comment:
Could you please drop the f in front of the field name (this would be the only place in Py4J where we do that) :-)